### ISSUE TITLE
some Mac binaries currently available in BioC 3.17 are broken

### ISSUE THREAD

**Issue by hpages** [Bioconductor/BBS#281]
@jwokaty Some Mac binaries currently available in BioC 3.17 are dynamically linked to `/opt/gfortran/lib/gcc/x86_64-apple-darwin20.0/12.2.0/libgfortran.5.dylib` and/or `/opt/gfortran/lib/gcc/x86_64-apple-darwin20.0/12.2.0/libquadmath.0.dylib`, so cannot be loaded on the end-user machine if they don't have gfortran installed. Some users have reported the problem already. See https://support.bioconductor.org/p/9151097/

To get the list of broken binaries, run the following on our Mac buider (e.g. on merida1):

1. Download the Intel Mac binaries available in BioC 3.17:
    ```
    rsync -e "ssh" -rptlzv --delete biocpush@nebbiolo1:PACKAGES/3.17/bioc/bin/macosx/big-sur-x86_64/contrib/4.3/ published_tgz
    ```
2. Extract them:
    ```
    mkdir extracted_tgz
    cd extracted_tgz
    for tgz in ../published_tgz/*.tgz; do echo -n "extracting $tgz ... "; tar zxf $tgz; echo "ok"; done
    ```
3. Check them:
    ```
    for dot_so_file in */libs/*.so; do otool -L $dot_so_file | grep '/opt/gfortran/' > /dev/null; if [ $? -eq 0 ]; then echo "$dot_so_file is affected"; fi ; done
    ```
This produces a list of 61 packages.

It would be good to run the same test for the arm64 Mac binaries in BioC 3.17 (with small adjustments to the above code).

I suspect that the trailing slash here: https://github.com/Bioconductor/BBS/blob/d6af06e973d56032970e98d5a91a119caf223e49/utils/macosx-inst-pkg.sh#L27 and here: https://github.com/Bioconductor/BBS/blob/d6af06e973d56032970e98d5a91a119caf223e49/utils/macosx-inst-pkg.sh#L30 breaks the following line: https://github.com/Bioconductor/BBS/blob/d6af06e973d56032970e98d5a91a119caf223e49/utils/macosx-inst-pkg.sh#L146 in the `macosx-inst-pkg.sh` script that we use to produce the `.tgz` tarballs during the BUILD BIN stage.

H.

**Comment by jwokaty** [comment]
The initial fix for this has been applied with #280 and bumping affected packages; however, this information seems valuable enough to document somewhere, possibly at the end of https://github.com/Bioconductor/BBS/blob/devel/Doc/Prepare-macOS-Big-Sur-HOWTO.md or in a separate document.

**Comment by hpages** [comment]
I think the fix is commit 89961932a44a913b775176c6e720ef163cb5ce23. Issue #280 is a separate issue.
